_data/simple-icons.json

```
@@ -3694,6 +3694,11 @@
                 ]
             }
         },
+        {
+            "title": "Grab",
+            "hex": "00B14F",
+            "source": "https://en.wikipedia.org/wiki/File:Grab_(application)_logo.svg)"
```

My apologies, something went wrong in my previous suggestion :sweat_smile: 

```suggestion
            "source": "https://en.wikipedia.org/wiki/File:Grab_(application)_logo.svg"
```